### ISSUE TITLE
New Card Counter not counting total new cards learned today

### ISSUE THREAD

**Issue by guesswho1234555** [AnKing-VIP/ankiweb-placeholder#107]
## Which add-on is this about?
New Card Counter

#### Problem description
Add-
*Please describe the issue concisely in here. In case of an error: Walk us through the steps you took to get there. What happened? What did you expect to happen?*

I used Anki to study for STEP 1 and had almost all my cards unsuspended. After the exam, I suspended all the cards and used the “Forget” feature in Anki. Now that I'm studying again, I'm unsuspending the cards I reset with the “Forget” feature.

My problem is that I relied on the “New Cards Counter” add-on to track my progress. However, this add-on does not show the cards I reset with “Forget” in the “Total new cards learned today” row. Only completely new cards that I've never seen or reset with “Forget” are displayed.

I don't want to reinstall the whole deck because I've added a lot of information in the Lecture field. What should I do?

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x ] I've restarted Anki to see if it helps
 - [x ] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [ ] I've tried to disable other add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [x ] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?
Version ⁨23.12.1 
*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

![image](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/173288972/a1f1f3b5-bfe9-451f-885f-e51c1af7da1b)
![add ons](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/173288972/bde27543-3eee-4ce6-b4bb-6b2bca415791)
![config](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/173288972/59c4de5d-b343-4c54-8e5f-21d390d32fe2)
![home page](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/173288972/ebef225e-25d3-4753-83fe-4cff880698a7)

**Comment by damronatron** [comment]
I just tried this with Anki 24.04.01. I deleted whatever version of the addon I had and reinstalled v0.2 from the AnKing VIP website. 
I did 3 cards. 1 new card that I have never done before and 2 cards that I previously studied and reset. The New Card Counter only registered the 1 new card that I had never done before. I even tried resetting a card, suspending it, then unsuspending it, and nothign changed. 
![CleanShot 2024-06-20 at 13 58 43@2x](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/54161626/d1886620-d81f-4ae5-b81c-be2773771ca1)

**Comment by AnKingMed** [comment]
I think it's because the addon uses Anki's built in "first time". @abdnh is there any way around this?

**Comment by abdnh** [comment]
Yes, this is due to how Anki's `introduced` query work. Fixing this appears to be tricky and will affect performance.

@damronatron If you don't care about losing review history, you can use the [Remove card history](https://ankiweb.net/shared/info/2089200096) add-on as a workaround.

**Comment by damronatron** [comment]
> Yes, this is due to how Anki's `introduced` query work. Fixing this appears to be tricky and will affect performance.
> 
> @damronatron If you don't care about losing review history, you can use the [Remove card history](https://ankiweb.net/shared/info/2089200096) add-on as a workaround.

I used that addon to remove my card history and then did a card and the New Card Counter did register it. Although anki has to do a full sync anytime you remove the card history, but this does seem to be an acceptable work around.

**Comment by AnKingMed** [comment]
@abdnh would it be better to address this on Anki's side?

**Comment by guesswho1234555** [comment]
Thank you for helping figure out a work around. Hope that this is something that can get patched more fully in the future

**Comment by abdnh** [comment]
> would it be better to address this on Anki's side?

Probably, not sure whether the performance would be acceptable though.
